src/FontAwesome5.WinUI/FontAwesome5.WinUI.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+		<TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+		<RootNamespace>FontAwesome5.WinUI</RootNamespace>
+		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+		<UseWinUI>true</UseWinUI>
+		<DefineConstants>WINUI</DefineConstants>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<DocumentationFile>$(OutDir)\FontAwesome5.WinUI.xml</DocumentationFile>
+		<Authors>Martin Topfstedt</Authors>
+		<PackageLicenseExpression>MIT</PackageLicenseExpression>
+		<Copyright>Copyright © Codinion 2018</Copyright>
+		<Company>Codinion</Company>
+	</PropertyGroup>
+
+	<ItemGroup>
```

Comment:
hi, i tested the code and i didn't saw any icons, i had to change the font files from Embedded Resource to Content and Copy Always, after that it worked.

Reply:
That is weird. Maybe because some cached files in obj/bin folders, it worked on my end.

Also, ironically the day after I submitted this PR, version 1.0 of the Windows App SDK has been released. Instead of the "project reunion" extension, a different extension by the "final" name, is needed.

I will update this PR will updates to Windows App SDK.

Reply:
I tested it both ways, and it seems to work using 'Content' and CopyAlways. I set it as you recommended. Thanks.